_runners/vault.py

```diff
@@ -72,14 +83,18 @@ def _get_policies(minion_id, config):
   #mappings = { 'minion': minion_id, 'grains': __grains__, 'pillar': minion_pillar }
   mappings = { 'minion': minion_id, 'grains': grains or {}}
 
+  my_policies = _get_my_policies()
```

Comment:
I think we should do this as part of initialization, rather than on every call, to keep the number of requests to Vault down.